migrations/2023-03-15-185959_add_dispute_table/up.sql

```diff
@@ -0,0 +1,35 @@
+CREATE TYPE "DisputeStage" AS ENUM ('pre_dispute', 'dispute', 'pre_arbitration');
+
+CREATE TYPE "DisputeStatus" AS ENUM ('dispute_opened', 'dispute_expired', 'dispute_accepted', 'dispute_cancelled', 'dispute_challenged', 'dispute_won', 'dispute_lost');
+
```

Comment:
Create INDEX for queryable fields

Reply:
Both DisputeStage and DisputeStatus are not queryable fields.